extensions/amp-story/1.0/amp-story.js

```diff
@@ -946,7 +946,12 @@ export class AmpStory extends AMP.BaseElement {
         );
 
         if (shouldReOpenAttachmentForPageId === this.activePage_.element.id) {
-          this.activePage_.openAttachment(false /** shouldAnimate */);
+          this.activePage_.element
+            .querySelector(
+              'amp-story-page-attachment, amp-story-page-outlink, amp-story-shopping-attachment'
+            )
+            ?.getImpl()
```

Comment:
You probably want `waitForUpgradedElement` here

Reply:
Also, I think somewhere in `amp-story-page` there's a string exactly with this value, you can make that string a constant and use it in both places.

Reply:
Do you mean `whenUpgradedToCustomElement`?